doc-content/enterprise-only/fuse/kie-camel-con.adoc

```diff
@@ -0,0 +1,60 @@
+[id='kie-camel-con_{context}']
+= The `kie-camel` component
+
+The `kie-camel` component is an Apache Camel endpoint provided by {FUSE_LONG} that integrates {FUSE} with {PRODUCT}.  It enables you to specify a {PRODUCT} module by using a Maven group ID, artifact, and version (GAV)  which you can pull into the route and execute. It also enables you to specify portions of the message body as facts.You can use the `kie-camel` component with embedded engines or with {KIE_SERVER}.
+
+.Embedded engines
```

Comment:
Instead of labels (".Label"), might consider this ascii sectioning alternative, which displays the label in bolded and indents the contents. You'll see what I mean if you build it. I use it frequently for cases like this.

```
facts.

You can use the `kie-camel` component with embedded engines or with {KIE_SERVER}:

kie-camel component in embedded engines::
In this scenario, ...
+
--
text 

text 
--

kie-camel component in {KIE_SERVER}::
In this scenario, ...
+
--
text

text
--
```

Or bulleted list with bolded labels:

```
facts.

You can use the `kie-camel` component with embedded engines or with {KIE_SERVER}:

* *kie-camel component in embedded engines:* In this scenario, ...
+
--
text 

text 
--
* *kie-camel component in {KIE_SERVER}:* In this scenario, ...
+
--
text

text
--
```

It's very subjective and strictly my opinion, but my rationale is that we typically use labels to label the content, such as Procedure, Figure, etc., and sometimes as labels for notes. In this case, a discrete sub-heading might work since it's a concept module, but the two segments aren't quite strong enough to merit it, which is why I agree that a label is better than a heading. But I'm just wondering of one of these others might be more suited. Completely your call.